frontend/src/modules/logs/constants.ts

```diff
@@ -7,23 +7,51 @@ import { PERCENT, LMIN, BPM } from '../info/units';
 import { EventType } from './EventType';
 
 export const eventDetailsMap = new Map<LogEventCode, string>([
-  [LogEventCode.mcu_backend_connection_down, 'Hardware controller lost connection'],
+  // System settings & alarms
+  [
+    LogEventCode.mcu_backend_connection_down,
+    'Hardware controller lost connection from backend server',
+  ],
   [
     LogEventCode.backend_mcu_connection_down,
     'Backend server lost connection from hardware controller',
   ],
-  [LogEventCode.backend_frontend_connection_down, 'User interface lost connection'],
-  [LogEventCode.frontend_backend_connection_down, 'User interface lost connection'],
-  [LogEventCode.mcu_backend_connection_up, 'Hardware controller connected'],
-  [LogEventCode.backend_frontend_connection_up, 'User interface connected'],
-  [LogEventCode.charger_disconnected, 'Battery charger is disconnected'],
+  [
+    LogEventCode.backend_frontend_connection_down,
+    'Backend server lost lost connection from user interface',
+  ],
+  [
+    LogEventCode.frontend_backend_connection_down,
+    'User interface lost connection from backend server',
+  ],
+  [LogEventCode.mcu_backend_connection_up, 'Hardware controller connected to backend server'],
+  [LogEventCode.backend_mcu_connection_up, 'Backend server connected to hardware controller'],
+  [LogEventCode.backend_frontend_connection_up, 'Backend server connected to backend server'],
```

Comment:
was curious about this, saw this while testing! is this supposed to be `Backend server connected to User interface` or am I misinterpreting something?